services/core-web/src/components/Forms/incidents/AddIncidentReportingForm.js

```diff
@@ -20,29 +28,37 @@ export const AddIncidentReportingForm = (props) => (
       <Row gutter={48}>
         <Col>
           {props.initialValues.mine_incident_id_year && (
-            <h4>{`Ministry Incident No. :  ${props.initialValues.mine_incident_report_no}`}</h4>
+            <h4>{`Ministry Incident No. ${props.initialValues.mine_incident_report_no}`}</h4>
           )}
           {props.initialValues.mms_inspector_initials ? (
             <span style={{ float: "right" }}>
-              {`MMS Inspector Initials: ${props.initialValues.mms_inspector_initials}`}
+              {`MMS Inspector Initials ${props.initialValues.mms_inspector_initials}`}
             </span>
           ) : (
             ""
           )}
           <Form.Item>
             <Field
-              id="mine_incident_category_code"
-              name="mine_incident_category_code"
-              label="Incident type*"
-              component={renderConfig.SELECT}
-              data={props.incidentCategoryCodeOptions}
+              id="categories"
+              name="categories"
+              label="Incident type(s)*"
+              placeholder="Select the incident type(s)"
+              component={renderConfig.MULTI_SELECT}
+              validate={[requiredList]}
+              data={
+                props.incidentCategoryCodeOptions
+                  ? props.incidentCategoryCodeOptions.sort((a, b) =>
+                      a.display_order > b.display_order ? 1 : -1
+                    )
+                  : []
+              }
```

Comment:
Display the incident category options sorted in ascending order by their display_order attribute.